support_bundle/get_support_bundle.sh

```diff
@@ -195,6 +195,9 @@ main() {
     echo "Using namespace ${NAMESPACE}"
     echo "Using context ${CONTEXT}"
 
+    # Collect kubectl cluster dump
+    kubectl ${KUBE_OPTS} cluster-info dump >> ${LOG_DIR}/kubectl-cluster-dump.json
```

Comment:
I gave a try to this command (I didn't know it, thanks for making me learn :grin: ) and is not only printing json structures but also al the logs of the containers in `kube-system` and `default` namespaces.
If you specify the `--output-directory` params it also create a nice hierarchy in the specified directory.

So I would suggest to do something like:
```suggestion
    CLUSTER_DUMP_DIR="${LOG_DIR}/kubectl-cluster-dump"
    mkdir ${CLUSTER_DUMP_DIR}
    kubectl ${KUBE_OPTS} cluster-info dump --output-directory=${CLUSTER_DUMP_DIR}
```

But please, feel free to ignore this comment is unnecessary!

Reply:
@patrickhargett I like this idea as well from @gmarraff